misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -360,8 +361,11 @@ public FunctionDefinitionNode transform(FunctionDefinitionNode functionDefinitio
             functionDefinitionNode = functionDefinitionNode.modify()
                     .withMetadata(metadata).apply();
         }
+
+        int startColumn = qualifierList.size() == 0 ? getStartColumn(functionDefinitionNode, true) : 0;
```

Comment:
shall we use isEmpty instead of checking size equals to zero

Reply:
Fixed